mixer/adapter/svcctrl/reportbuilder.go

```diff
@@ -306,3 +306,12 @@ func (b *reportBuilder) generateAPIResourceLabels() map[string]string {
 	labels["cloud.googleapis.com/location"] = "global"
 	return labels
 }
+
+func newReportBuilder(instance *svcctrlreport.Instance, supportedMetrics []metricDef,
```

Comment:
Please test this function - mainly to improve code coverage.

Reply:
done